test/testwave/testfiles/t_2_027.cpp

```diff
@@ -0,0 +1,13 @@
+/*=============================================================================
+    Boost.Wave: A Standard compliant C++ preprocessor library
+    http://www.boost.org/
+
+    Copyright (c) 2001-2012 Hartmut Kaiser. Distributed under the Boost
+    Software License, Version 1.0. (See accompanying file
+    LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+=============================================================================*/
+
+// Test error reporting during redefinition of '__has_include'
+
```

Comment:
Does this miss a `//O --c++20`?

Reply:
It will be `//O --c++17`... probably sometime later today.